support/@trustfractal/data-host/.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode",
+    "eamodio.gitlens",
+    "streetsidesoftware.code-spell-checker",
+  ]
+}
```

Comment:
I personally don't use vscode. It also seems strange to include editor specific config in source control. I'd prefer not to have this unless there's a good reason to.

Reply:
I am using vscode and thought it would be useful, but I can add it as a local configuration.